docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -31,6 +31,9 @@ Software
 - A web browser.
 - An up-to-date clone of the `Flocker repository <https://github.com/ClusterHQ/flocker.git>`_.
 - An up-to-date clone of the `homebrew-tap repository <https://github.com/ClusterHQ/homebrew-tap.git>`_.
+- `gsutil <https://pypi.python.org/pypi/gsutil>`_
```

Comment:
This should be installed via the `release` extra.

I guess perhaps this is for outside of a VM. In which case https://clusterhq.atlassian.net/browse/FLOC-1036 would be the correct solution.